Algorithms/Java/SortingTechniques/BubbleSort/BubbleSortEnhanced.java

```diff
@@ -12,7 +12,7 @@
  * (n is the array size)
  *
  * No. of passes = (n-1)
- * No. of comparisions = 1 + 2 + 3+.....n-1= n(n-1)/2 [O(n^2)]
```

Comment:
Mere ko IELTS ka score check krna hai tera bhai xD

Reply:
Typo tha bro, I have refactored it in other classes.